fs/backgroundfetcher/resolver.go

```diff
@@ -82,15 +86,21 @@ func (lr *sequentialLayerResolver) Resolve(ctx context.Context) (bool, error) {
 		"spanId": lr.nextSpanFetchID,
 	}).Debug("fetching span")
 
+	if lr.nextSpanFetchID == 0 {
+		lr.start = time.Now()
+	}
 	err := lr.FetchSingleSpan(lr.nextSpanFetchID)
 	if err == nil {
+		commonmetrics.IncOperationCount(commonmetrics.BackgroundSpanFetchCount, lr.layerDigest)
```

Comment:
I think this metric should really be part of the bg-fetcher, not resolver.

Reply:
Why? ``Resolver`` is part of the bg-fetcher